third_party/git-cmake-format/CMakeLists.txt

```diff
@@ -1,7 +1,12 @@
-message(STATUS "Configuring and building git-cmake-format")
-ginkgo_load_git_package(git-cmake-format
-    "https://github.com/ginkgo-project/git-cmake-format.git"
-    "be9554a5d71030dcc23f1eb355c41ab9d876e776"
-    "-DGCF_CLANGFORMAT_MINIMAL_VERSION=8.0.0")
-add_subdirectory(${CMAKE_CURRENT_BINARY_DIR}/src
-    ${CMAKE_CURRENT_BINARY_DIR}/build EXCLUDE_FROM_ALL)
+include(FetchContent)
+FetchContent_Declare(
+    git_cmake_format
+    GIT_REPOSITORY https://github.com/ginkgo-project/git-cmake-format.git
+    GIT_TAG        be9554a5d71030dcc23f1eb355c41ab9d876e776
+)
+FetchContent_GetProperties(git_cmake_format)
+if(NOT git_cmake_format_POPULATED)
+    message(STATUS "Fetching git-cmake-format")
```

Comment:
Shouldn't the message be at the top level? e.g., before the `FetchContent_Declare`. Also to uniformize with the other packages.

Reply:
Good point, that was an oversight